### DISCUSSION TITLE
Adding Cyrillic Letters

### DISCUSSION THREAD

**Discussion by thejupitergroup** [ivodolenc/aspekta#18] [Ideas]
### Request Description

Could you add Cyrillic letters to expand the language possibility of Aspekta?

### Alternative Solutions

We've tried other fonts, but we don't feel they do the job like Aspekta.

**Comment by ivodolenc** [comment]
Hi, thanks for your interest in Aspekta, maybe in future releases as this request takes a lot of time. 

I'm open for PRs, if anyone is interested in helping out.

**Comment by ivodolenc** [comment]
Related #25
